packages/jetpack-redirects/test/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Internal dependencies
+ */
+import getRedirectUrl from '../src';
+
+describe( 'getRedirectUrl', () => {
+	test( 'simple url', () => {
```

Comment:
The test descriptions usually describe in one line what the test expects, and often form a sentence with the parent `describe` string. The test runner will concatenate them together. Example:
```
getRedirectUrl › returns URL with source=simple param
getRedirectUrl › ignores an unknown, unsupported param in args
getRedirectUrl › encodes special characters in params correctly
... etc.
```